frontend/ui/widget/inputtext.lua

```diff
@@ -643,6 +643,8 @@ function InputText:onKeyPress(key)
         handled = true
         if key["Back"] then
             self:delChar()
+        elseif key["Del"] then
+            self:delWord()
```

Comment:
Dunno if we need to be consistent with Del and Shift+Del acting on the following chars - Shift+Del with this will delete previous word I guess.

Reply:
could make it `key["Del"] and hasSym()`

Reply:
Would this be consistent with the other specific different handling on Kindle you added previously?
(If it is just for _you_ to have this on _your_ Kindle because you like it, and not bother us on the emulator, it does not make it less inconsistent :) and I'd rather have the same consistent mapping - or lack of a mapping - on all devices, and not a special case on Kindle because you want it that way.)
(I also don't know if Sym = Shift on a Kindle, or if it is a Kindle only modifier key.)

Reply:
`Sym` is a button that only exists on kindle (the keyboard variants), it is **not** the same as `Shift`. For some reason it is defined as a mod (but I don't think it should be one though).

well `Del` on kindle functions as backspace, so `Shift` + `Del` as deleting a whole word does not seem inconsistent (to me at least) but if you disagree well... let the odds be forever in your favour ;)

Reply:
> well `Del` on kindle functions as backspace, so `Shift` + `Del` as deleting a whole word does not seem inconsistent (to me at least)

Well, not disagreeing then :)